moninag/static/src/ts/app/app-header/app-header.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="flex-container" fxLayout="row" fxLayoutAlign="space-between start">
+    <div id="left-block" class="flex-container" fxLayout="row" fxLayoutAlign="start center">
+        <a class="flex-container" fxLayout="row" fxLayoutAlign="start center" href="/">
+            <img id="logo-img" src="./static/img/logo.png"/>        
+            <h1 id="title">{{ title }}</h1>
+        </a>
+    </div>
+
+    <div *ngIf="user" id="user-block">        
+        <button md-button [mdMenuTriggerFor]="menu">
+            <img alt="User picture" src="https://www.gravatar.com/avatar/{{ user.avatar }}">
+            <span>{{ user.first_name }} {{ user.second_name }}</span>            
+        </button>
+        <md-menu x-position="before" #menu="mdMenu">
+            <button md-menu-item (click)="viewProfile()">
+                <md-icon>account_circle</md-icon>
+                <span>Profile</span>
+            </button>
+            <button md-menu-item (click)="logOut()">
+                <md-icon>exit_to_app</md-icon>
+                <span>Log Out</span>
+            </button>
+        </md-menu>
+    </div>
+</div>
```

Comment:
Because it is like small pop up menu, and no different content to display like tabs :slightly_smiling_face: 
It is done like in Trello :eyes: